CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Update fluentd to 1.12.2-sumo-10
```

Comment:
PR link 😿

Reply:
Too late, brutally merged by @perk-sumo while I wasn't looking. I'll add it tomorrow.